bool resume_spiral_motion
---
bool result